perception/traffic_light_arbiter/README.md

```diff
@@ -0,0 +1,36 @@
+# traffic_light_arbiter
+
+## Purpose
+
+This package receives traffic signals from perception and V2X components and combines them using either a confidence-based or a V2X-preference based approach.
+
+## TrafficLightArbiter
+
+A node that merges traffic light/signal state from image recognition and V2X to provide a planning component.
+It's currently a provisional implementation.
+
+### Inputs / Outputs
+
+#### Input
+
+| Name                             | Type                                         | Description                                              |
+| -------------------------------- | -------------------------------------------- | -------------------------------------------------------- |
+| ~/sub/vector_map                 | autoware_auto_mapping_msgs/msg/HADMapBin     | The vector map to get valid traffic signal ids.          |
+| ~/sub/perception_traffic_signals | autoware_perception_msgs::msg::TrafficSignal | The traffic signals from the image recognition pipeline. |
```

Comment:
```suggestion
| ~/sub/perception_traffic_signals | autoware_perception_msgs/msg/TrafficSignal | The traffic signals from the image recognition pipeline. |
```

Reply:
Actually, `types` use `::` unlike include paths that use `/` .
Please refer to other packages (e.g., https://github.com/autowarefoundation/autoware.universe/tree/main/perception/lidar_centerpoint). 
Still, some of the other topics were wrong, so I fixed them in [271fcc0](https://github.com/autowarefoundation/autoware.universe/pull/3780/commits/271fcc0a9110feee40e8f53bc6e93cfdf0b06db1)